mysql-test/suite/rocksdb.sys_vars/t/rocksdb_compact_cf_basic.test

```diff
@@ -1,3 +1,6 @@
+
+call mtr.add_suppression(" Column family '[a-z]*' not found.");
```

Comment:
Should go after the have_foo checks (not necessarily actionable in the current PR)